build/parseSpec.c

```diff
@@ -1118,3 +1146,25 @@ rpmSpec rpmSpecParse(const char *specFile, rpmSpecFlags flags,
 {
     return parseSpec(specFile, flags, buildRoot, 0);
 }
+
+rpmRC parseGeneratedSpecs(rpmSpec spec)
+{
+    
+    ARGV_t argv = NULL;
+    int argc = 0;
+    int i;
+
+    char * specPattern = rpmGenPath("%{u2p:%{_builddir}}", spec->buildSubdir, "__rpmbuild_*.specpart");
```

Comment:
"__rpmbuild_*.specpart" still looks quite the eyesore to me. These are not rpm internal things, but files we expect others to produce. I'd think "*.specpart" would be enough to differentiate from other stuff.

Also there's still the option of creating a special directory (".rpmbuild" or such in the builddir) which could be used for such things, including but not limited to debuginfo filelists, to avoid cluttering the main build directory (that gets requested separately every now and then). Such a thing doesn't have to be part of this PR, but should need to be done before a public release.